Jenkinsfile

```
@@ -108,6 +108,7 @@ pipeline {
 						set +e
 						while [ $ready -ne 0 ]; do
 							curl --fail --verbose http://127.0.0.1:9901/api/v2/blocks
+							curl --fail --verbose http://127.0.0.1:9901/api/v2/accounts
```

Why did we shift from `/api/v2/blocks` to `/api/v2/accounts`?

We should change to `/api/ready` instead. But this would only be doable after we finish: #262 or add dummy transactions after the container build phase during the CI.
```suggestion
							curl --fail --verbose http://127.0.0.1:9901/api/ready
```

---

I guess that waiting for accounts (or transactions) gives us more time to build an index during initial start. The `/api/ready` endpoint reports gateway readiness only.

---

Then, perhaps waiting for `/api/v2/votes` is the best check to have here as it would ensure that all the indexes have started building.

---

I think for now `/api/v2/accounts` is enough as we dont have any transactions available in `Jenkins`. Once #372 is implemented, we can enable check either for `votes` or `transactions`.